### ISSUE TITLE
[FEATURE] Support multiple datasources for the "Authentication" tab

### ISSUE THREAD

**Issue by derek-ho** [opensearch-project/security-dashboards-plugin#1797]
**Is your feature request related to a problem?**
Support multiple datasources functionality in the "Authentication" tab. This should be a relatively small task since this tab mainly read-only information about the authentication methods of the current datasource

**Acceptance criteria**
- Data source picker shared component is present on the page with a sensible UX
- Data source selections that should be persisted across tabs is accurately persisted
- Selected datasource prior to landing on this page is correctly populated (if applicable), and defaults are selected if not applicable
- We are showing the authentication and authorization information for the correct (currently selected) datasource

**What solution would you like?**
Support multiple datasources in the "Authentication" tab. 

**What alternatives have you considered?**
None

**Do you have any additional context?**
Depends on #1795 to have data source picker available.

**Comment by derek-ho** [comment]
[Triage] This is part of the meta of supporting multi datasources in the security plugin.

**Comment by derek-ho** [comment]
Merged into feature branch
